packages/adapters/subgraph/.graphclientrc.yml

```diff
@@ -537,9 +537,14 @@ sources:
   - name: StableSwap_Optimism
     handler:
       graphql:
-        endpoint: https://api.thegraph.com/subgraphs/name/connext/stableswap-v0-optimism
-        retry: 5
-        timeout: 30000 # 30 seconds
+        strategy: fallback
+        sources:
+          - endpoint: "https://connext.bwarelabs.com/subgraphs/name/connext/stableswap-v0-optimism"
+            retry: 5
+            timeout: 30000 # 30 seconds
```

Comment:
I updated stableswap subgraph to track lp token transfers. 
https://github.com/connext/monorepo/pull/4274

So we need to deploy stable swap subgraphs again, after merging into prod, I will deploy to thegraph.com